doc/nrf/ug_pelion.rst

```diff
@@ -0,0 +1,121 @@
+   The Device Management Update link points to the documentation of the version compatible with the |NCS|.
+   For details, refer to the `Pelion Device Management documentation`_ and `Pelion Device Management release notes`_.
+
+Create a dedicated patition on flash
+====================================
+
+The Pelion Device Management library requires a partition named ``pelion_storage`` to be defined on flash.
+The partition is used to store the non-volatile data, such as credentials and identifiers.
+Because this kind of data is fragile, keep the partition on the SoC internal flash or make sure to secure the access to the memory.
+
+Configure dynamic allocation memory
+===================================
+
+The Pelion Device Management library depends on the new libc.
+In case of targets without the memory management unit (MMU), given that the application is not running in the Userspace, the portion of RAM that remains unallocated for any other purpose is used by the libc heap.
+Make sure to leave a sufficient space for heap so the Pelion Device Management library may work correctly.
+
+For more information about C standard library refer to :ref:`libc_api`.
+
+Enable and configure the network backend
+========================================
+
+The Pelion Device Management library uses standard POSIX sockets for the network communication.
+The library was tested with the following network backends:
+
+* Cellular (LTE, NB-IoT)
+* OpenThread
+
+After initializing and setting up the Pelion object instance, it continuously retries to connect to the Pelion's Device Management server.
+
+Cellular backend
+----------------
+
+Cellular backend makes use of the on board modem being part of the nNR9160 SiP.
+For more information about working with the modem please refer to :ref:`ug_nrf9160`.
+
+To simplify the connection setup the LTE link controller can be used (see :ref:`lte_lc_readme`).
+
+OpenThread backend
+------------------
+
+Thread is a low-power mesh networking technology.
+I allows a device to access the Internet if one of the mesh elements (boarder router) share the Internet connection.
+
+For more information please refer to :ref:`zephyr:thread_protocol_interface` and :ref:`ug_thread`.
+
+Additional configuration
+************************
+
+You can control the Pelion Device Management library features using Kconfig options that are defined within the Pelion Device Management library repository.
+
+The following options are among the most important ones:
+
+* :option:`CONFIG_PELION_UPDATE` - This option enables the device firmware update (DFU) feature.
+* :option:`CONFIG_PELION_TRANSPORT_MODE_TCP`, :option:`CONFIG_PELION_TRANSPORT_MODE_UDP`, :option:`CONFIG_PELION_TRANSPORT_MODE_UDP_QUEUE` - These options select the transport protocol used by the library.
+
+For more information and the full list of the library configuration options, read the `Zephyr integration tutorial`_ in the Pelion documentation.
```

Comment:
Looks like the link is not valid (https://developer.pelion.com/docs/device-management/current/connecting/zephyr.html). It gives me 404. Also searching for Zephyr in the Pelion docs site did not give any results.

Reply:
This is the link provided by @marcuschangarm and it will be populated by the release time.